### DISCUSSION TITLE
Calling an asset from a graph

### DISCUSSION THREAD

**Discussion by yuval-airis** [dagster-io/dagster#18317] [Q&A] [answered]
I am using an asset-based approach and had several assets calling each one:

@asset 
def func1(s: int):
   ...

@asset 
def func2(func1):
   ... 
   
etc.

Now I created a new graph and wanted to call func1 from it. Something like:

@graph 
def some_graph():
   func1(s=5)
    ....

Running the graph I don't see any errors, but it doesn't appear that func1 is called at all.
Is this the wrong way to make that call?

Do I have to pass it as some sort of config instead of a param? (Since otherwise it'd expect it to come from another op?)

**Comment by smackesey** [comment, marked as answer]
Please see the [Asset concept page](https://docs.dagster.io/concepts/assets/software-defined-assets) for how to assemble assets into a graph (you don't do this using `@graph`, that is for ops).
